rider-fsharp/protocol/src/kotlin/model/RdFSharpTypeProvidersLoaderModel.kt

```diff
@@ -0,0 +1,157 @@
+package model
+
+import com.jetbrains.rider.model.nova.ide.SolutionModel
+import com.jetbrains.rd.generator.nova.*
+import com.jetbrains.rd.generator.nova.PredefinedType.*
+import com.jetbrains.rd.generator.nova.csharp.CSharp50Generator
+
+@Suppress("unused")
+object RdFSharpTypeProvidersLoaderModel : Root(CSharp50Generator(
+        FlowTransform.Symmetric,
+        "JetBrains.ReSharper.Features.WinForms.Designer.Protocol.Model")
+) {
+    private val Type = structdef {

```

Comment:
`RdProvidedType`?
You then won't have to do stuff like `using Type = System.Type;`.

Reply:
Fixed